src/jdbc-auth.md

```diff
@@ -1435,59 +1405,49 @@ descriptive name with the protocol to be used for the connection. It contains
+  unique within the same connection group. Other tables reference connections
+  using `connection_id`, not `connection_name`.
 
 `protocol`
-: The protocol to use with this connection. This is the name of the protocol
-  that should be sent to guacd when connecting, for example "`vnc`" or 
-  "`rdp`".
+: The protocol for the connection (e.g., "`vnc`" or "`rdp`").
 
 `parent_id`
 : The unique integer associated with the connection group containing this 
   connection, or `NULL` if this connection is within the root group.
 
 `max_connections`
-: The maximum number of concurrent connections to allow to this connection 
-  at any one time *regardless of user*. `NULL` will use the default value 
-  specified in `guacamole.properties` and a value of `0` denotes unlimited.
+: The maximum concurrent connections allowed to this connection. `NULL` uses the
+  default value from `guacamole.properties`, and `0` denotes unlimited.
 
 `max_connections_per_user`
-: The maximum number of concurrent connections to allow to this connection  
-  at any one time *from a single user*. `NULL` will use the default value  
-  specified in `guacamole.properties` and a value of `0` denotes unlimited.
+: The maximum concurrent connections allowed per user. `NULL` uses the default
+  value from `guacamole.properties`, and `0` denotes unlimited.
 
 `proxy_hostname`
-: The hostname or IP address of the Guacamole proxy daemon (guacd) which 
-  should be used for this connection. If `NULL`, the value defined with the 
-  `guacd-hostname` property in `guacamole.properties` will be used.
+: The hostname or IP address of the Guacamole proxy daemon (guacd) for this
+  connection. If `NULL`, the `guacd-hostname` property in
+  `guacamole.properties` will be used.
 
 `proxy_port`
-: The TCP port number of the Guacamole proxy daemon (guacd) which should be 
-  used for this connection. If `NULL`, the value defined with the 
-  `guacd-port` property in `guacamole.properties` will be used.
+: The TCP port number of the Guacamole proxy daemon (guacd) for this connection.
+  If `NULL`, the `guacd-port` property in `guacamole.properties` will be used.
 
 `proxy_encryption_method`
-: The encryption method which should be used when communicating with the 
-  Guacamole proxy daemon (guacd) for this connection. This can be either 
-  `NONE`, for no encryption, or `SSL`, for SSL/TLS. If `NULL`, the 
-  encryption method will be dictated by the `guacd-ssl` property in 
-  `guacamole.properties`.
+: The encryption method for communicating with the Guacamole proxy daemon
+  (guacd): `NONE` for no encryption or `SSL` for SSL/TLS. If `NULL`, the
+  `guacd-ssl` property in `guacamole.properties` will be used.
 
 `connection_weight`
-: The weight for a connection, used for applying weighted load balancing 
-  algorithms when connections are part of a `BALANCING` group. This is an 
-  integer value, where values `1` or greater will weight the connection 
```

Comment:
That this should be an integer seems relevant and probably shouldn't be cut from the docs.